lib/query.js

```diff
@@ -2951,11 +2951,14 @@ Query.prototype.distinct = function(field, conditions, callback) {
  *     // equivalent
  *     query.sort('field -test');
  *
+ *     // also possible is to use a array with array key-value pairs
+ *     query.sort([["field", "asc"]]);
+ *
  * #### Note:
  *
  * Cannot be used with `distinct()`
  *
- * @param {Object|String} arg
+ * @param {Object|String|Array.<Array<(string | number)>>} arg
```

Comment:
Is the extra dot a typo?
```suggestion
 * @param {Object|String|Array<Array<(string | number)>>} arg
```

Reply:
not that i know of, though https://jsdoc.app/tags-type.html , though i should either remove it, or add it to the second array too

Reply:
changed to be consistent with 8bbcd93f715e13c656691e4224287bbbd74edc7a